app/views/merchant/orders/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<%@order.item_orders.each do |item_order|%>
+  <section id='item_order-<%=item_order.item_id %>'>
+    Item Id: <%=item_order.item_id%><br>
+    Item Name: <%=item_order.item.name%><br>
+    From: <%=item_order.item.merchant.name%><br>
```

Comment:
I fee we can get rid of some of these long AR changes by calling a different instance variable
For this view